spec/untangled/client/core_spec.cljs

```diff
@@ -107,6 +107,52 @@
       (assertions
         (get-in @state [:many :path]) => [[:table 99] [:table 3] [:table 77]]))))
 
+(specification "integrate-ident"
+  (let [state {:a {:path [[:table 2]]}
+                  :b     {:path [[:table 2]]}
+                  :d     [:table 6]
+                  :many  {:path [[:table 99] [:table 88] [:table 77]]}
+                  }]
+    (behavior "Can append to an existing vector"
```

Comment:
In untangled-spec you can actually do this more simply:
```
(assertions
    string
    expr => expr
    ...
    string
    expr => expr
   ...)
```

In other words, the acceptable syntax for assertions body is:

`(string? triple+)+`